mmv1/third_party/terraform/services/bigtable/resource_bigtable_instance.go

```diff
@@ -793,6 +818,14 @@ func resourceBigtableInstanceClusterReorderTypeListFunc(diff tpgresource.Terrafo
 				return fmt.Errorf("Error setting cluster diff: %s", err)
 			}
 		}
+
+		oNsf, nNsf := diff.GetChange(fmt.Sprintf("cluster.%d.node_scaling_factor", i))
+		if oNsf != nNsf {
+			err := diff.ForceNew(fmt.Sprintf("cluster.%d.node_scaling_factor", i))
+			if err != nil {
+				return fmt.Errorf("Error setting node scaling factor: %s", err)
+			}
+		}
```

Comment:
Instead of using diff.ForceNew function, `ForceNew:    true,` can be added for the field.

Reply:
Done